### PR TITLE
changes "print" and "download" to "create"

### PR DESCRIPTION
**Pull request by esotrope**
screen shot 2017-10-10 at 1 25 06 pm

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -445,20 +445,19 @@
...
+          "multiple": "Create {{name}} Student Reports"
         },
-        "regenerate": "Regenerate",
         "assessment-type": "Assessment Type",
         "subject": "Subject",
         "school-year": "School Year",
         "accommodations": {
           "title": "Student Accommodations",
-          "info": "TODO"
+          "info": ""
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: @jtreuting realized we still don't have text for the info button above student accommodations on the report form. Until we have the text i'm tempted to take the info button out.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -514,7 +513,7 @@
         "submit": {
           "label": "Export Results",
           "success-html": "Creating export. See <a href=\"/reports\">status</a>.",
-          "failure": "Error creating report. Please try again."
+          "failure": "Error creating export. Please try again."
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: unrelated - this is to change the org level export fail message.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 3]
```diff
@@ -445,20 +445,19 @@
...
       "form": {
         "title": {
-          "single": "Print {{firstName}}'s Report",
-          "single-prepopulated": "Print {{name}}'s {{schoolYear}} {{subject}} {{assessmentType}} Report",
-          "multiple": "Print {{name}} Student Reports"
+          "single": "Create {{firstName}}'s Report",
+          "single-prepopulated": "Create {{name}}'s {{schoolYear}} {{subject}} {{assessmentType}} Report",
+          "multiple": "Create {{name}} Student Reports"
         },
-        "regenerate": "Regenerate",
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: This isn't used when a report fails?

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Right now it just shows an ```fa-refresh``` icon instead

**Pull request merged**